src/System.CommandLine/System.CommandLine.csproj

```diff
@@ -13,6 +13,8 @@
     * Constructor- and property-based model binding for complex types
     * Test and debug support
     </Description>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- <NoWarn>$(NoWarn);CS1591</NoWarn> -->
```

Comment:
Do we need this NoWarn?

Reply:
I guess not. I'm surprised the build didn't fail on account of this warning.